config/sources.inc.php

```diff
@@ -715,12 +715,12 @@
                 'nl', 'pl', 'pt-PT', 'ro', 'sk', 'sl', 'sv-SE',
             ],
         'firefox/android/index.lang'            => $android_locales,
-        'firefox/australis/firefox_tour.lang'   => $mozillaorg,
-        'firefox/australis/fx36_tour.lang'      => $mozillaorg,
-        'firefox/channel.lang'                  => $mozillaorg,
-        'firefox/desktop/customize.lang'        => $mozillaorg,
-        'firefox/desktop/fast.lang'             => $mozillaorg,
-        'firefox/desktop/index.lang'            => $mozillaorg,
+        'firefox/australis/firefox_tour.lang'   => $firefox_desktop,
+        'firefox/australis/fx36_tour.lang'      => $firefox_desktop,
+        'firefox/channel.lang'                  => $firefox_desktop,
```

Comment:
We should probably keep this one as `$mozillaorg´, since it has download buttons for Firefox for Android too. I agree on all the other pages.